packages/malloy-render/.storybook/preview-head.html

```diff
@@ -0,0 +1,2 @@
+<link rel="preconnect" href="https://rsms.me/" />
+<link rel="stylesheet" href="https://rsms.me/inter/inter.css" />
```

Comment:
Wondering if we should package this CSS in our repo somehow so that rendering our CSS doesn't require connecting to their server.

Reply:
Inter is our suggested font for using Malloy, but we don't actually ship it as part of the bundled renderer. It is up to the developer using the renderer to load the font themselves if they want to use it. This is similar to what Google Material 3 does, where Roboto is their default font but it is on the app developer to bring that font in https://github.com/material-components/material-web/blob/main/docs/theming/typography.md

In the code snippet above, that is only executed as part of the Storybook that we use to preview our components, so that won't get shipped as part of Malloy renderer.

In terms of where to recommend loading the font from, it is also available on Google Fonts https://fonts.google.com/specimen/Inter?query=inter. I believe I loaded it directly from source in the Storybook because the latest version had just been released and I wanted to try some of the new number variants.